bigbone/src/main/kotlin/social/bigbone/api/method/Media.kt

```diff
@@ -1,16 +1,25 @@
 package social.bigbone.api.method
 
+import okhttp3.MediaType.Companion.toMediaTypeOrNull
 import okhttp3.MultipartBody
+import okhttp3.RequestBody.Companion.asRequestBody
 import social.bigbone.MastodonClient
 import social.bigbone.MastodonRequest
 import social.bigbone.api.entity.Attachment
 
 class Media(private val client: MastodonClient) {
-    //  POST /api/v1/media
-    fun postMedia(file: MultipartBody.Part): MastodonRequest<Attachment> {
+    /**
+     * Creates an attachment to be used with a new status.
+     * @param file the file that should be uploaded
+     * @param mediaType media type of the file as a string, e.g. "image/png"
+     * @see <a href="https://docs.joinmastodon.org/methods/media/#v1">Mastodon API documentation: methods/media/#v1</a>
+     */
+    fun postMedia(file: java.io.File, mediaType: String): MastodonRequest<Attachment> {
```

Comment:
Can we import java.io.File here?

Reply:
Stupid mistake from reusing the method signature as discussed in 62. Of course, and fixed! :)